doc/man3/DEFINE_STACK_OF.pod

```diff
@@ -90,18 +92,28 @@ B<NULL>.
 sk_TYPE_value() returns element B<idx> in B<sk>, where B<idx> starts at
 zero. If B<idx> is out of range then B<NULL> is returned.
 
-sk_TYPE_new() allocates a new empty stack using comparison function B<compar>.
-If B<compar> is B<NULL> then no comparison function is used.
+sk_TYPE_new() allocates a new empty stack using comparison function B<compare>.
+If B<compare> is B<NULL> then no comparison function is used. This function is
+equal to sk_TYPE_new_reserve(compare, 0).
```

Comment:
I wonder if "is equal to" is right here. English speakers?

Reply:
I would tend to say "is equivalent to"

Reply:
It's the same…… https://english.stackexchange.com/questions/71259/is-equal-to-or-equals

Reply:
Question is not about "equals" vs. "to be equal to", but rather about applicability of "equality" itself to function implementations. Values can be equal, values returned by functions can be equal, but functions themselves ... gut feeling not so much. I'd even take it further by saying that "equality" applies rather to *arithmetic* values, not values in general. For example, I'd be inclined to say "same pointer as" rather than "pointer equals to"...

Reply:
Nah, I just look this up in a dictionary and it explains as:

http://www.dictionary.com/browse/equal

Note the 'the same as' explanation. So you can exchange 'the same as' and 'equals to'...

Reply:
equivalent is a better word.